src/patternfly/demos/DataTable/data-table-expandable-table.hbs

```diff
@@ -77,7 +77,7 @@
     {{#> table-tr}}
       {{#> table-td table-td--toggle="true" table-td--button--attribute=(concat 'aria-labelledby="' table--id '-node2 expandable-toggle2" id="expandable-toggle2" aria-label="Details" aria-controls="' table--id '-content2"')}}{{/table-td}}
       {{#> table-td table-td--check="true"}}
-        {{#> check-input check-input--attribute=(concat 'type="checkbox" name="checkrow2" aria-labelledby="' table--id '-node2" aria-label="Details"')}}{{/check-input}}
+        <input type="checkbox" name="checkrow2" aria-labelledby="{{table--id}}-node2">
       {{/table-td}}
       {{#> table-th table-th--data-label="Repository Name"}}
         <div id="{{table--id}}-node2">Node 2<div><a href="#">siemur/test-space</a></div></div>
```

Comment:
Not sure if this is relative to this PR, but the structure we have in the component example is different:
```
<div>
  <div id="simple-table-with-checkboxes-node2">Node 2</div>
  <a href="#">siemur/test-space</a>
</div>
```
In the component example, the html illustrates how the the id is only on the meaningful portion of the `<th>` if there are a lot of contents included there.

Reply:
thanks! updated the structure.